.pubnub.yml

```diff
@@ -192,8 +192,8 @@ sdks:
                   is-required: Required
                 -
                   name: okhttp
-                  min-version: 3.12.6
-                  location: https://repo.maven.apache.org/maven2/com/squareup/okhttp3/okhttp/3.12.6/okhttp-3.12.6.jar
+                  min-version: 4.10.0
```

Comment:
Can you please also fix (remove) the notice about OkHttp 3.x here: https://www.pubnub.com/docs/sdks/kotlin#download-the-sdk